### PR TITLE
administrative unit data

### PR DESCRIPTION
**Pull request by atrisovic**
addresses #42

### CODE REVIEW

File: tutorial-content/content/geographical-unit-data.md [hunk 1]
```diff
@@ -1,79 +1,80 @@
...
-data format is an ESRI Shapefile, and we will focus on these for this
-tutorial. You may also encounter GDB (common in hydrology), GeoJSON
-(used with web interfaces), KXML (used by Google Earth), and OSM (from
-OpenStreetMap) files. All of these can be converted to Shapefiles,
+Administrative units are often defined and described as shapefiles. A [shapefile](https://www.esri.com/library/whitepapers/pdfs/shapefile.pdf) stores geometry and attribute information for the spatial features in a data set. The geometry for a feature is stored as a shape comprising a vector of coordinates. Area features are represented as closed-loop, double-digitized polygons. The shapes together with data attributes linked to each shape create the representation of geographic data like countries, rivers and lakes.
+ 
+Describing shapes requires specialized data formats. The most common data format is an ESRI Shapefile, and we will focus on these for this tutorial. You may also encounter GDB (common in hydrology), GeoJSON (used with web interfaces), KXML (used by Google Earth), and OSM (from OpenStreetMap) files. All of these can be converted to Shapefiles,
```
Comment (jrising) [inline tutorial-content/content/geographical-unit-data.md:58]: I think this should go above the preceding paragraph.

**Review by jrising — APPROVED** [review]
I only skimmed this, but it looks like a good improvement. I'll read through everything again before we re-submit.

File: tutorial-content/content/geographical-unit-data.md (hunk 1 above)
Comment (atrisovic) [reply tutorial-content/content/geographical-unit-data.md:58]: I saw it as: first we introduce what a shape file is (line 56), and then we talk about more technical things like shape file formats (line 58).

Comment (jrising) [reply tutorial-content/content/geographical-unit-data.md:58]: Unfortunately, I don't think there's a name for the class of information that is distinct from it's format. "Shapefile" means a file of the ESRI Shapefile format. How about reword the first paragraph to not use the term "shapefile"?

```
The boundaries of administrative units are described in specialized files that stores geometry and attribute information. The geometry for a feature is represented by a vector of coordinates....
```

Comment (atrisovic) [reply tutorial-content/content/geographical-unit-data.md:58]: Ah ok, I understand now what you mean - I didn't know that the term shapefile, that now people use so often, started with the format itself 😄 Ok I'll make this change!

Comment (atrisovic) [reply tutorial-content/content/geographical-unit-data.md:58]: Also, what should I put as decision questions for this section? I can also leave it empty for now

Comment (atrisovic) [reply tutorial-content/content/geographical-unit-data.md]: Added these two for the section:

```
- How to incorporate geographical and administrative units in your analysis?
- How to generate weather measures that will correspond to your economic data and geographical regions?
```

**Pull request merged**